src/blockchain_utilities/blockchain_stats.cpp

```diff
@@ -203,16 +210,20 @@ plot 'stats.csv' index "DATA" using (timecolumn(1,"%Y-%m-%d")):4 with lines, ''
   }
   std::cout << ENDL;
 
+#define MAX_INOUT	32
+#define MAX_RINGS	65000
+
   struct tm prevtm = {0}, currtm;
   uint64_t prevsz = 0, currsz = 0;
   uint64_t prevtxs = 0, currtxs = 0;
   uint64_t currblks = 0;
   uint64_t totins = 0, totouts = 0, totrings = 0;
   boost::multiprecision::uint128_t prevemission = 0, prevfees = 0;
   boost::multiprecision::uint128_t emission = 0, fees = 0;
-  uint32_t minins = 10, maxins = 0;
-  uint32_t minouts = 10, maxouts = 0;
-  uint32_t minrings = 50, maxrings = 0;
+  boost::multiprecision::uint128_t totdiff = 0, mindiff = 0, maxdiff = 0;
+  uint32_t minins = MAX_INOUT, maxins = 0;
+  uint32_t minouts = MAX_INOUT, maxouts = 0;
+  uint32_t minrings = MAX_RINGS, maxrings = 0;
```

Comment:
Why not just 0xffffffff or std::numeric_limits<uint32_t>::max() ?

Reply:
Didn't seem necessary at the time, and the latter is just too long to type every time. But sure, I can use one of those in the #define instead.